software.md

```diff
@@ -18,7 +18,7 @@ Here you can find links to software related to RPM such as frontends, packaging
 
 ### Packager tools
 * [rpmlint](https://github.com/rpm-software-management/rpmlint) is a tool for checking for common errors in RPM packaging.
-* [rpmdevtools](https://hosted.fedoraproject.org/projects/rpmdevtools) contains scripts and (X)Emacs support files to aid in RPM packaging.
+* [rpmdevtools](https://fedoraproject.org/wiki/Rpmdevtools) contains scripts and (X)Emacs support files to aid in RPM packaging.
```

Comment:
Shouldn't we instead link to https://pagure.io/rpmdevtools ?

Reply:
Yes, but the page is so barren. The Fedora page has at least some content.

Reply:
OK makes sense, and there's also a link to the GH repo on that page, after all.